server/chart_config.json

```diff
@@ -153,11 +181,25 @@
             "statsVars": [
               "Count_Person_Upto5Years",
               "Count_Person_5To17Years",
+              "Count_Person_15To19Years",
```

Comment:
ditto

Reply:
Done

Reply:
this is how it looks like: https://screenshot.googleplex.com/UKW2QQGQOTj

we do need to fix too many years though...

Reply:
iirc, census has all of these, we just left a bunch out from these charts (you'll notice that it's not actually a complete range, 18-24 is missing.

![image](https://user-images.githubusercontent.com/6052978/87637471-1cc33b80-c6f7-11ea-82d1-bfd037da7a68.png)

Reply:
Let's see how it looks like across countries. I suspect we'll have to remove most of these new categories quickly..

Reply:
removed two more... we can trim afterwards..

Reply:
can you stamp again?